DependencyInjection/Configuration.php

```diff
@@ -50,6 +50,12 @@ public function getConfigTreeBuilder()
                         ->scalarNode('password')->end()
                     ->end()
                 ->end()
+                ->arrayNode('html5validation')
+                    ->children()
+                        ->scalarNode('url')->end()
+                        ->arrayNode('ignores_extract')->end()
```

Comment:
Anyway you're right, the `html5validation` node is not documented in the project's README, I'll try to improve this and submit a PR.